paper_sections/05-computational_and_statistical_issues.Rmd

```diff
@@ -526,16 +527,14 @@ be extracted with:
 ```{r brms_code, echo=T, eval =F, message=F, warning=F}
 stancode(CO2_mod2_brms)
 ```
-
-**should we mention gretagam at this point? and does INLAbru have any capacity for HGAMs? I'm not familiar enough with INLAbru's inner workings. I do know that glmmTMB cannot currently include mgcv-style smooths.**
 -->
 
-<!--## Beyond HGAMs: functional regression
```

Comment:
I think this is an important section to include, as anyone who wants to go much further with these models than what we described here will have to move to full functional regression. Also, I didn't want it to look like we were ignoring this work, or weren't aware of it.

Reply:
I agree that this is important and should go in the paper, I just think it goes in the discussion rather than this section.